students/andrew_garcia/lesson02-assignment/charges_calc.py

```diff
@@ -0,0 +1,118 @@
+"""
+Returns total price paid for individual rentals
+"""
+
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def debug_setup(debug_level):
+    """ Sets the logging file name and levels """
```

Comment:
One alternative is to add these two lines here:

```
if debug_level == 0:
    logger.disabled = True
    return
```

Note that the return statement means the function will stop running code there. You can then setup the file handler outside of the subsequent if's, except for setting the corresponding level.